src/main/config/config_system.h

```diff
@@ -20,6 +20,11 @@
 typedef struct systemConfig_s {
     uint8_t emf_avoidance;                   // change pll settings to avoid noise in the uhf band
     uint8_t i2c_highspeed;                   // Overclock i2c Bus for faster IMU readings
+#ifdef CUSTOM_FLASHCHIP
+    uint32_t flashchip_id;
+    uint16_t flashchip_nsect;
+    uint16_t flashchip_pps;
+#endif
 } systemConfig_t;
```

Comment:
this doesn't belong here.  create a new parameter group specifically for it.

https://en.wikipedia.org/wiki/Separation_of_concerns

Developers shouldn't have to change system configuration code to make changes to flash chip configuration information.